Branch-SDK/src/io/branch/indexing/BranchUniversalObject.java

```diff
@@ -324,14 +353,24 @@ public boolean isPublicallyIndexable() {
         return indexMode_ == CONTENT_INDEX_MODE.PUBLIC;
```

Comment:
fyi, I know this is already an existing function, bht publically is not correct spelling - the correct spelling (and it's this way in v2/event) is publicly

Reply:
Will make it with next API onwards